wallet/how-to/connect/access-accounts.md

```diff
@@ -118,3 +124,11 @@ function handleAccountsChanged(accounts) {
 Previously, `eth_accounts` returned at most one account in the `accounts` array.
 The first account in the array will always be considered the user's "selected" account.
 :::
+
+## Revoke access to a user's accounts
+
+Since `eth_requestAccounts` internally calls `wallet_requestPermissions` for permission to call
+`eth_accounts`, you can use [`wallet_revokePermissions`](/wallet/reference/wallet_revokePermissions)
+to revoke this permission, revoking your dapp's access to the user's accounts.
```

Comment:
```suggestion
to revoke this permission, revoking your dapp's access to the user's accounts. This can be useful as a more convenient way for users to logout (or disconnect) from your site. You can then rely on the permissions that MetaMask reports as a way to indicate whether they are logged in / logged out (or connected / disconnected) of your site.
```